annotations/src/main/java/org/p3model/annotations/technology/TierAttribute.java

```diff
@@ -0,0 +1,10 @@
+package org.p3model.annotations.technology;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Target;
+
+@Target(ElementType.MODULE)
```

Comment:
Question is, which Java JDK versions we support (8 or 11 or 17 LTS)?

The `ElementType.MODULE` is not inherited in Java 8 (https://docs.oracle.com/javase/8/docs/api/java/lang/annotation/ElementType.html), 
but for v11 it is already (https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/annotation/ElementType.html).

According to state of [Java ecosystem in 2023](https://newrelic.com/resources/report/2023-state-of-the-java-ecosystem) the JDK 11 is now most widely used (56% versus 32% for JDK 8). I would stay with v11.

What do you think?